### ISSUE TITLE
Cannot install faiss-gpu on Arch Linux

### ISSUE THREAD

**Issue by TheYellowArchitect** [kyamagu/faiss-wheels#89]
**Describe the bug**
I am trying to install faiss-gpu.

**To Reproduce**

```
pip install faiss-gpu
```
outputs
```
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement faiss-gpu (from versions: none)
ERROR: No matching distribution found for faiss-gpu
```

I even tried to download the .whl file from `https://pypi.org/project/faiss-gpu/#files`  and input
```
pip install --use-wheel --no-index --find-links=[FILEPATH] faiss-gpu
```

**Expected behavior**
Normal installation like any other pip package

**Desktop (please complete the following information):**
 - OS: Artix Linux
 - Architecture: x64
 - Python: 3.11 but also tried with pyenv  -> pyenv global 3.9.8. Same results
 - Version: 1.7.2

**Additional context**
I am trying to download/install this, in order to `make install` this https://github.com/Mangio621/Mangio-RVC-Fork
In the above's `makefile` there is this line basically `	pip install faiss-gpu fairseq gradio ffmpeg ffmpeg-python praat-parselmouth pyworld numpy==1.23.5 numba==0.56.4 librosa==0.9.1` and it fails at the very first, others have no problem if i run them by themselves.

**Comment by kyamagu** [comment]
Check the README. Source package is also not available at this moment.

>Note GPU package has been supported until version 1.7.2, but is not available since version 1.7.3 due to https://github.com/kyamagu/faiss-wheels/issues/57.

**Comment by TheYellowArchitect** [comment]
Makes sense. I am looking forward to the above being fixed
